README.md

```diff
@@ -62,26 +63,31 @@ Please don't hesitate to open an issue in case you encounter any issue due to po
 
 ## Getting started
 
-The [`notebooks` directory](https://github.com/owkin/PyDESeq2/blob/main/notebooks/README.md) contains minimal examples on how to use PyDESeq2, in the form of jupyter notebooks.
+The [Getting Started](https://pydeseq2.readthedocs.io/en/latest/auto_examples/index.html) section of the documentation contains downloadable examples on how to use PyDESeq2.
 
-You can also try them from your browser (on synthetic data only):  
+You can run them directly in your browser:  
 
-[![Binder](https://mybinder.org/badge_logo.svg)](https://mybinder.org/v2/gh/owkin/PyDESeq2/HEAD?labpath=notebooks%2Findex.ipynb) 
+[![Binder](https://mybinder.org/badge_logo.svg)](https://mybinder.org/v2/gh/Owkin/PyDESeq2/5bf8ec61831abda43ab472517bf62c1d176b5043?urlpath=lab/tree/jupyter_notebooks/auto_examples/plot_minimal_pydeseq2_pipeline.ipynb) 
```

Comment:
I would remove it unless you are 100% sure this link won't change with new versions? I would guess that it does

Reply:
Indeed, good point. Let me remove it.